test/starlark_tests/targets_under_test/ios/BUILD

```diff
@@ -1061,6 +1061,24 @@ ios_extension(
 
 # ---------------------------------------------------------------------------------------
 
+ios_application(
+    name = "app_with_codesignopts",
+    bundle_id = "com.google.example",
+    codesignopts = ["--digest-algorithm=sha1", "--digest-algorithm=sha384"],
```

Comment:
should we add the resource rules case as a test here to validate the expansion behavior works?